### ISSUE TITLE
Export from ON gives errors "Could not find a part of the path" (pages with trailing spaces)

### ISSUE THREAD

**Issue by Bratieres** [alxnbl/onenote-md-exporter#93]
**Describe the bug**
When exporting certain notes, onenote md exporter ignores those with message "Could not find a part of the path " followed by path name

**To Reproduce**

Create a ON note with one or several " " at the end of note or section title

**Expected behavior**

Ignore trailing spaces in note title or section title
**Logs**

[18:20:22 WRN] Page 'Informatique\Installations\Installations\Sauvegarde Mes Documents': Erreur durant le traitement de la page [f909c7e356c349a9825d7a34898eb465]. Page ignorée. Message d'erreur : Could not find a part of the path 'C:\Users\jb\Desktop\ONMdexporter\Exports\md\Informatique-20230708 18-19\Informatique\Installations\Installations\Rueil Sauvegardes \Sauvegarde Mes Documents.md'. (space after Rueil Sauvegardes)

**Desktop (please complete the following information):**
 - OneNodeMdExporter version :  ?
 - Windows version : 11 32/64 bits
 - Office Version : XXXX 32/64 bits

**Additional context**

**Comment by alxnbl** [comment]
Thank you for the bug report and the reproduction steps. I will correct it in the next version.

**Comment by alxnbl** [comment]
Hi @Bratieres , could you send me a sample Onenote notebook that I can use to reproduce the problem  please ?
I did not managed to reproduce the bug by myself.

**Comment by Bratieres** [comment]
Well, I tried to recreate the bug by adding spaces after notes, section, ... titles, but all do export fine. Maybe my diagnosis is wrong. I noted the " " before \ in folder name and supposed this was the problem. I removed the " " in note and it worked. Now if I add the " " , export works. Maybe " " was another character that coud not be displayed (may be <tab>) ?

I probably have some backups of my notes somewhere if this can help you.
